sklearn/utils/tests/test_utils.py

```diff
@@ -7,6 +7,7 @@
 import pytest
 import numpy as np
 import scipy.sparse as sp
+from pandas.core.common import SettingWithCopyWarning
```

Comment:
We can not import pandas at the top level because `pandas` is an optional dependence.